zoom_monitor.py

```diff
@@ -80,14 +81,39 @@ def _get_raw_url(url):
         # directories in the path to skip that.
         return f"https://app.zoom.us/wc/join/{url.split('/')[-1]}"
 
+    async def _wait_for_meeting_start(self, just_joined: bool):
+        """
+        Wait for the meeting to start.
+
+        Args:
+            just_joined (bool): If the bot just joined the meeting.
+        """
+        if just_joined:
+            await asyncio.sleep(5)
+        waiting_message_locator = self._driver.get_by_text(
+            "waiting for the host to start"
+        )
+        waiting_message_count = await waiting_message_locator.count()
+        if waiting_message_count == 0:
+            self._meeting_started = True
+        else:
+            self._logger.info("meeting hasn't started")
+            await asyncio.sleep(30)
```

Comment:
This feels like it would be sluggish. My first instinct is that we should sleep for at most 1 second on line 92, and at most 10 seconds here. but you'd know better than me: why are these times so long?